package.json

```diff
@@ -27,6 +27,11 @@
       "types": "./dist/types/plugins/new-relic-plus/index.d.ts",
       "import": "./dist/esm/plugins/new-relic-plus/index.js",
       "default": "./dist/cjs/plugins/new-relic-plus/index.js"
+    },
```

Comment:
You'll also need to bump the package version to release a new version of the library. You can run `npm version minor` and commit the results!

Reply:
Done!